### PR TITLE
New package: rofi-wayland-1.7.5+wayland1

### PR DESCRIPTION
**Pull request by inalone**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by classabbyamp** [comment]
previous prs: #33578 #40102 #42064

**Comment by inalone** [comment]
I've read over instances before in which this package was denied due to either
a) the PR being stale
b) maintainers being unconvinced of its usefulness

As a user of both wofi and rofi, rofi's ecosystem is fantastic. I can easily just search up Rofi scripts, some of which really old, some of which well-established, some of which package right here on Void! Using the Wayland friendly rofi "replacements" is not the same experience, even though those projects are great. Rofi is just such a well established project and ecosystem.

File: srcpkgs/rofi-wayland/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'rofi-wayland'
+pkgname=rofi-wayland
+version=1.7.5
+revision=1
+_ver=${version}+wayland1
+wrksrc=rofi-${_ver}
```
Comment (classabbyamp) [inline srcpkgs/rofi-wayland/template:6]: this variable is read-only now, do not set it

File: srcpkgs/rofi-wayland/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'rofi-wayland'
+pkgname=rofi-wayland
+version=1.7.5
+revision=1
+_ver=${version}+wayland1
```
Comment (classabbyamp) [inline srcpkgs/rofi-wayland/template:5]: if this is part of the version, put it in the version

File: srcpkgs/rofi-wayland/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+makedepends="gdk-pixbuf-devel libxkbcommon-devel pango-devel
+ wayland-protocols wayland-devel"
+checkdepends="check-devel cppcheck xkeyboard-config"
+short_desc="Window switcher, run dialog and dmenu replacement (Wayland fork)"
+maintainer="inalone <me@inal.one>"
+license="MIT"
+homepage="https://github.com/lbonn/rofi"
+distfiles="https://github.com/lbonn/rofi/releases/download/${_ver}/rofi-${_ver}.tar.gz"
+checksum=4b7b5de0606e91380861dce6f2b894bebf63b2031c8746fd4096f72576ce541a
+replaces="rofi>=0"
```
Comment (classabbyamp) [inline srcpkgs/rofi-wayland/template:19]: this should probably be conflicts

**Review by classabbyamp — COMMENTED** [review]
still mightily don't like this. if rofi's devs wanted to support wayland, they'd support wayland.

**Comment by inalone** [comment]
> still mightily don't like this. if rofi's devs wanted to support wayland, they'd support wayland.

I completely agree with this, and I think most of the Rofi community has respected the Rofi devs' right to do that. I think however that this fork is of such popularity + with more people moving off Xorg that a large amount of people will be running this on their Void machines regardless, so it should be packaged (albeit obviously different to the upstream version)

**Pull request closed without merging**